Formula/docker-ls.rb

```diff
@@ -0,0 +1,38 @@
+class DockerLs < Formula
+  desc "Tools for browsing and manipulating docker registries"
+  homepage "https://github.com/mayflower/docker-ls"
+  url "https://github.com/mayflower/docker-ls.git",
+    :tag => "v0.3.1",
+    :revision => "d80310976c9707e261e57ebfa9acf4e0b1781460"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    (buildpath/"src/github.com/mayflower/docker-ls").install buildpath.children
+
+    cd "src/github.com/mayflower/docker-ls" do
+      system "go", "build", "-o", bin/"docker-ls", "./cli/docker-ls"
+      system "go", "build", "-o", bin/"docker-rm", "./cli/docker-rm"
+      prefix.install_metafiles
+    end
+
+    system "go", "generate", "github.com/mayflower/docker-ls/lib"
```

Comment:
You should be able to get rid of this line & everything below it until the `end` now?

Reply:
Sorry, my bad, I've removed the old code. However, I can't get rid of the `go generate` step; this is required to generate versioning information